playwright.config.js

```diff
@@ -0,0 +1,61 @@
+// @ts-check
+const {defineConfig, devices} = require('@playwright/test');
+
+/**
+ * @see https://playwright.dev/docs/test-configuration
+ */
+module.exports = defineConfig({
+    testDir: './tests/e2e',
+
+    /* Run tests in files in parallel */
+    fullyParallel: true,
+
+    /* Fail the build on CI if you accidentally left test.only in the source code. */
+    forbidOnly: !!process.env.CI,
+
+    /* Retry on CI only */
+    retries: process.env.CI ? 2 : 0,
+
+    /* Opt out of parallel tests on CI. */
+    workers: process.env.CI ? 1 : undefined,
+
+    /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+    reporter: 'html',
+
+    /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+    use: {
+        /* Base URL to use in actions like `await page.goto('/')`. */
+        baseURL: 'http://127.0.0.1:3000',
+
+        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+        trace: 'on-first-retry',
+    },
+
+    /* Configure projects for major browsers */
+    projects: [
+        {
+            name: 'chromium',
+            use: {...devices['Desktop Chrome']},
+        },
+
+        {
+            name: 'firefox',
+            use: {...devices['Desktop Firefox']},
+        },
+
+        /*
+        {
+            name: 'webkit',
+            use: {...devices['Desktop Safari']},
+        },
+        */
```

Comment:
If we need it, let's add a comment how to use it.
```suggestion
```

Reply:
What do you mean? To use it is just to comment out this. Right now is commented because there's an issue with running with multiple browser contexts in WebKit.

Reply:
Ah, okay, I meant if you are going to merge PR with the commented out code, let's left a comment like: "there's an issue with running with multiple browser contexts in WebKit."

Reply:
Done 👍